packages/cli/src/cliEntry.js

```diff
@@ -138,11 +138,19 @@ async function run() {
 }
 
 async function setupAndRun() {
-  const setupEnvScript = /^win/.test(process.platform)
-    ? path.join('..', 'setup_env.bat')
-    : path.join('..', 'setup_env.sh');
-
-  childProcess.execFileSync(path.join(__dirname, setupEnvScript));
+  // We only have a setup script for UNIX envs currently
+  if (/^win/.test(process.platform) === false) {
+    const scriptName = 'setup_env.sh';
+    const absolutePath = path.join(__dirname, '..', scriptName);
+
+    try {
+      childProcess.execFileSync(absolutePath);
+    } catch (error) {
+      logger.warn(
+        `Failed to run environment setup script "${scriptName}". This script can continue to run if your local environment matches what React Native expects. If it does fail, check out "${absolutePath}" and adjust your environment to match it.\n${error}`
```

Comment:
Actually it seems like `This script can continue to run` may be confusing (what is "script"). Could you change it to `The react-native CLI continues to run` or something?

Reply:
Adjusted the wording and added red color to original error:

<img width="943" alt="screenshot 2019-02-20 at 09 02 43" src="https://user-images.githubusercontent.com/5106466/53075810-600f7f80-34ee-11e9-9c9e-0b10fb42e300.png">

Reply:
Awesome!